lib/Dredd.js

```diff
@@ -108,7 +144,7 @@ class Dredd {
       skipped: 0,
       start: 0,
       end: 0,
-      duration: 0,
+      duration: 0
```

Comment:
Not to keen on removing the trailing commas because then when you add new entries to objects/arrays you have to include modification of other lines in the diff, which is why it is in our style guide.

Reply:
Yes, completely agree. I've removed trailing commas due to Node6 compliance, but then I've realized that source code is not run in Node, the built version is. Will revert the trailing comma rule.

Reply:
Files formatting made consistent in b7911c81cfe0411bd399a9f5f6d43ee18e8c8492.